packages/miniflare/src/plugins/core/inspector-proxy/inspector-proxy-relay.ts

```diff
@@ -0,0 +1,161 @@
+import assert from "node:assert";
+import WebSocket from "ws";
+import { isDevToolsEvent } from "./devtools";
+import type {
+	DevToolsCommandRequests,
+	DevToolsEvent,
+	DevToolsEvents,
+} from "./devtools";
+
+export class InspectorProxyRelay {
```

Comment:
I must admit that I am not too convinced of this class name, I chose "relay" since this is relaying messages to specific workers (probably I should add some comments here...)

But I'm really not sure of it... maybe something like `WorkerInspectorProxy` would be better? 🤔

any thoughts?

Reply:
Or just `WorkerInspector`? TBH I think the current name is fine.

Reply:
I'm not sure about `WorkerInspector` since having `Proxy` somewhere in the name of the class seems quite valuable here (since essentially this is a proxy)

actually! maybe this should actually be called `InspectorProxy` (since it is proxying an inspector channel) while what I currently named `InspectorProxy` could be called `InspectorProxyController` (or something like that)

Meaning that we have a single `InspectorProxyController` that controls one or more `InspectorProxy`s 🤔 (one per worker)

I think that this would be more correct and would also make things a bit clearer to me... what do you think? 🤔

Reply:
I've updated the names to `InspectorProxyController` and `InspectorProxy` 🙂